PulseSampleApp/app/src/main/java/com/ooyala/sample/players/PulsePlayerActivity.java

```diff
@@ -120,4 +128,18 @@ public VideoItem getVideoItem() {
 		videoItem.setContentCode(getIntent().getExtras().getString("embedCode"));
 		return videoItem;
 	}
+
+	private void destroyPlayer() {
+		if (player != null) {
+			player.destroy();
+			player = null;
+		}
+		if (skinLayout != null) {
+			skinLayout.release();
```

Comment:
I look in skinLayout code and noticed that in constructor we use setOnSystemUiVisibilityChangeListener, but don't delete it in release. Maybe we can add deleting just to avoid issues in future?

Reply:
There is no `unregister` method for this purpose. Also since the fact that the view is attached to a parent so it will be destroyed as soon as the parent will.